python/tempo/tsdf.py

```diff
@@ -161,15 +161,15 @@ def __addPrefixToColumns(self, col_list: list[str], prefix: str) -> "TSDF":
 
         # build a column rename map
         col_map = {col: "_".join([prefix, col]) for col in col_list}
+        # In the future (when Spark 3.4+ is standard) we should implement batch rename using:
```

Comment:
```suggestion
        # TODO - In the future (when Spark 3.4+ is standard) we should implement batch rename using:
```

Adding a TODO so it's easier to find again :)